### ISSUE TITLE
Fast Boot

### ISSUE THREAD

**Issue by mrochaaa** [v1s1t0r1sh3r3/airgeddon#193]
Just an opinion, when you open airgeddon, it can check many dependences, and is slow, is good an fast boot like Wifite, not to put each script side by side, just an opinion, an check and splash (UFO up and down) in first time is excelent but not every day, maybe an option? --fastboot

**Comment by OscarAkaElvis** [comment]
Probably we'll develop in the future something like "silent checks" and maybe some skip intro option in options menu or something similar. Thanks for the suggestion. Closing.

**Comment by xtonousou** [comment]
Maybe we can use ENV variables and place them in some .conf files.

```bash
export AIRGEDDON_CHECK_DEPENDENCIES=0
export AIRGEDDON_CHECK_UPDATES=0
export AIRGEDDON_COLORS=0
export AIRGEDDON_HINTS=0
```

**Comment by OscarAkaElvis** [comment]
That's a good approach. Probably setting a default values on airgeddon.sh in order to avoid errors if for any reason the flags.env can't be loaded (one thing less to control).

We (the developers) can discuss this for the future v9.0 version which is now on enterprise branch. This is not hard to implement.

**Comment by OscarAkaElvis** [comment]
This is already implemented. It will be released on the future v9.0.

Documentation about possible options is here: https://github.com/v1s1t0r1sh3r3/airgeddon/wiki/Options

As you can see, there are a lot of available options to skip intro, disable auto update, disable colors, disable print hints, silent dependency checks, etc.

Closing this.
